cx.modal.Reflection.json

```diff
@@ -0,0 +1,72 @@
+{
+    "id" : "cx.modal.Reflection",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "49",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "reflection",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--env=RUST_BACKTRACE=1",
+        "--env=RUST_LOG=reflection=INFO,reflection_doc=INFO,reflection_node=INFO"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin"
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+    	{
+    	    "name" : "libspelling",
+	    "buildsystem" : "meson",
+	    "sources" : [
+	    	{
+	    	    "type" : "git",
+	    	    "url" : "https://gitlab.gnome.org/GNOME/libspelling.git",
+	    	    "tag" : "0.4.9",
+		    "commit" : "e41e11d6ca6f5af987a46caa08036d5f0f629892"
+	    	}
+	    ]
+    	},
+        {
+            "name" : "reflection",
+            "buildsystem" : "meson",
+	    "config-opts" : [
+                "--libdir=lib"
+            ],
```

Comment:
```suggestion
```

Reply:
sorry, i guess i didn't push the right changes